tests/validation/tests/v3_api/test_bkp_restore.py

```diff
@@ -1,21 +1,34 @@
+from urllib.parse import urlparse
+
 import pytest
 from .common import *  # NOQA
+from .test_rke_cluster_provisioning import rke_config, engine_install_url, node_template_linode, \
+    validate_rke_dm_host_2
 
 namespace = {"p_client": None, "ns": None, "cluster": None, "project": None}
 
 
-def test_bkp_restore():
+def test_bkp_restore_local(create_project_client):
+    validate_backup_create_restore_delete("local")
+
+
+def test_bkp_restore_s3_with_creds(create_project_client_and_cluster_s3_with_creds):
+    validate_backup_create_restore_delete("s3")
+
+
+def test_bkp_restore_s3_with_iam(create_project_client_and_cluster_s3_with_iam):
+    validate_backup_create_restore_delete("s3")
+
+
+def validate_backup_create_restore_delete(backup_mode):
     p_client = namespace["p_client"]
     ns = namespace["ns"]
     client = get_user_client()
     cluster = namespace["cluster"]
     name = random_test_name("default")
 
-    if not hasattr(cluster, 'rancherKubernetesEngineConfig'):
-        assert False, "Cluster is not of type RKE"
-
```

Comment:
@izaac We will need this check to prevent the test cases from running in a non-rke cluster in the case of test_bkp_restore_local